hokusai/build.yml

```diff
@@ -5,3 +5,5 @@ services:
       context: ../
       args:
         - COMMIT_HASH
+    environment:
```

Comment:
Not necessary to set that here - can leave it to the default `3005` locally (environment variables are set only when starting a container - values set here won't persist)